tests/countries/test_taiwan.py

```diff
@@ -628,26 +639,33 @@ def test_taiwan_united_nations_day(self):
         self.assertNoHolidayName(name)
 
     def test_taiwan_retrocession_day(self):
-        name = "臺灣光復節"
+        name_old = "臺灣光復節"
+        name_new = "臺灣光復暨金門古寧頭大捷紀念日"
```

Comment:
This naming approach is not quite future-proof if this gets renamed one more time.